modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ObjectHandlerScript.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.servicetemplate;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import java.io.Serial;
+import java.io.Serializable;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+/** Defines the handler script of service object manage. */
+@Valid
+@Data
+@Slf4j
+public class ObjectHandlerScript implements Serializable {
```

Comment:
we do not need a new type for this. we can reuse ServiceChangeScript type.
only the field name in the OCL must be "objects"  specific.
I made a mistake in the OCL sample. Have corrected that also.